### DISCUSSION TITLE
部署正常，访问正常，但注册失败。重新创建应用部署后依然失败。

### DISCUSSION THREAD

**Discussion by nanbowanya** [walinejs/waline#419] [Q&A] [answered]
部署正常，访问正常，但注册失败。重新创建应用部署后依然失败。有没有大佬遇到过解决成功？
![](https://tva1.sinaimg.cn/large/008i3skNgy1gsjvjkojkqj31ku0u0gnc.jpg)

**Comment by nanbowanya** [comment]
日志提示
![](https://tva1.sinaimg.cn/large/008i3skNgy1gsjvr787h2j31940u049y.jpg)

**Comment by lizheming** [comment]
提供下问题地址，部署方式等相关信息我们帮忙排查一下。

**Reply by nanbowanya** [reply]
请问需要提供什么地址？是vercel的还是waline插件的能具体一点吗

**Reply by nanbowanya** [reply]
部署方式按照教程走的。https://waline.js.org/guide/get-started.html

**Reply by lizheming** [reply]
https://github.com/walinejs/waline/issues/new?assignees=&labels=&template=bug-report.md&title=%5BBUG%5D

**Reply by nanbowanya** [reply]
已提交

**Reply by wubaiqi** [reply]
我也是注册失败，请问你解决了吗，哪个环节出的问题

**Reply by nanbowanya** [reply]
> 我也是注册失败，请问你解决了吗，哪个环节出的问题

重新部署一遍就可以了

**Reply by tabris-trees** [reply]
> 我也是注册失败，请问你解决了吗，哪个环节出的问题
请问你解决了吗，我重新部署还是没有解决┭┮﹏┭┮

**Reply by tabris-trees** [reply]
> > 我也是注册失败，请问你解决了吗，哪个环节出的问题
> > 请问你解决了吗，我重新部署还是没有解决┭┮﹏┭┮

换成国际版的leadcloud之后可以了

**Comment by Mister-Hope** [comment, marked as answer]
请其他用户注意，vercel修改环境变量必须需要重新部署。此项在文档中也有提示。
